src/foam/u2/wizard/WAO.js

```diff
@@ -8,6 +8,9 @@ foam.INTERFACE({
   package: 'foam.u2.wizard',
   name: 'WAO',
   label: 'Wizardlet Access Object',
+  todo: [
```

Comment:
I was unaware of this axiom ;-).

Reply:
It's new; it's so new that it technically doesn't exist yet.

Reply:
While we could add it to `foam.core.Model` or a documentation-related refinement, my thinking is to leave it in this purgatory intentionally and decide where to put that when it becomes useful.

Reply:
FOAM1 had issues: []

https://github.com/foam-framework/foam/blob/9e8500f149625aebbc5dd5eabd722b1225ce3ce1/core/mm1Model.js#L829

Which was basically a cross between an issue tracker and more structured TODOs. Then you just use git as your issue tracker and create a view of the ModelDAO for viewing issues. Then the issue gets updated in the same PR as the code that fixes it.